src/containers/CartPage.jsx

```diff
@@ -92,6 +93,11 @@ class CartPage extends React.Component {
       })
         .then(res => res.json())
         .then((res) => {
+          if (res.links.length === 0 && !res._defaultcart) {
```

Comment:
The check `res.links.length === 0` is not triggered, so the message never gets displayed